converter/topas/parser.py

```diff
@@ -1,17 +1,72 @@
 from converter.common import Parser
-
+from pathlib import Path
 
 class DummmyParser(Parser):
-    """A simple placeholder parser that does nothing."""
+    """A simple placeholder parser that ignores the json input and prints example (default) configs."""
 
     def __init__(self) -> None:
-        raise NotImplementedError("Don't use me, i don't work yet :(")
+        self.config = """s:Ge/MyBox/Type     = "TsBox"
+s:Ge/MyBox/Material = "Air"
+s:Ge/MyBox/Parent   = "World"
+d:Ge/MyBox/HLX      = 2.5 m
+d:Ge/MyBox/HLY      = 2. m
+d:Ge/MyBox/HLZ      = 1. m
+d:Ge/MyBox/TransX   = 2. m
+d:Ge/MyBox/TransY   = 0. m
+d:Ge/MyBox/TransZ   = 0. m
+d:Ge/MyBox/RotX     = 0. deg
+d:Ge/MyBox/RotY     = 0. deg
+d:Ge/MyBox/RotZ     = 0. deg
+
+s:So/Demo/Type = "Beam" 
+s:So/Demo/Component = "BeamPosition"
+s:So/Demo/BeamParticle = "proton"
+d:So/Demo/BeamEnergy = 169.23 MeV
+u:So/Demo/BeamEnergySpread = 0.757504
+s:So/Demo/BeamPositionDistribution = "Gaussian" 
+s:So/Demo/BeamPositionCutoffShape = "Ellipse" 
+d:So/Demo/BeamPositionCutoffX = 10. cm 
+d:So/Demo/BeamPositionCutoffY = 10. cm 
+d:So/Demo/BeamPositionSpreadX = 0.65 cm 
+d:So/Demo/BeamPositionSpreadY = 0.65 cm 
+s:So/Demo/BeamAngularDistribution = "Gaussian" 
+d:So/Demo/BeamAngularCutoffX = 90. deg
+d:So/Demo/BeamAngularCutoffY = 90. deg
+d:So/Demo/BeamAngularSpreadX = 0.0032 rad 
+d:So/Demo/BeamAngularSpreadY = 0.0032 rad 
+i:So/Demo/NumberOfHistoriesInRun = 0
```

Comment:
Add output section, by adjusting this example:
https://topas.readthedocs.io/en/latest/examples-docs/MVLinac/WaterPhantom.html?highlight=Dose

Reply:
As parent use MyBox